pipeline/terraform/2023-09-29/main.tf

```diff
@@ -0,0 +1,30 @@
+module "pipeline" {
+  source = "../modules/stack"
+
+  reindexing_state = {
+    listen_to_reindexer      = false
+    scale_up_tasks           = false
+    scale_up_elastic_cluster = false
+    scale_up_id_minter_db    = false
+    scale_up_matcher_db      = false
+  }
+
+  index_config = {
+    works = {
+      identified = "works_identified.2023-05-26"
+      merged     = "works_merged.2023-05-26"
+      indexed    = "works_indexed.2023-09-12"
+    }
+    images = {
+      indexed        = "images_indexed.2023-09-12"
+      works_analysis = "works_indexed.2023-05-26"
```

Comment:
I don't quite understand why it's not using the new version here, ie `works_indexed.2023-09-12`.

Reply:
That's a mistake, but it's OK, because they are actually identical.  I created the new one because it was simpler for the Works (which currently only specify one date)